environment.yml

```diff
@@ -19,5 +19,6 @@ dependencies:
   - geoalchemy2
   - pip:
     - geofabrics>=0.9.3
+    - geovoronoi==0.4.0
```

Comment:
```suggestion
    - geovoronoi>=0.4.0
```

Reply:
Would it be better if I changed this?

Reply:
I think that would be fine for now since we have not locked down most of the other packages yet.